README.md

```diff
@@ -99,20 +102,23 @@ trie.get(address, function (err, data) {
 
   console.log('------Storage------')
   var stream = storageTrie.createReadStream()
-  stream.on('data', function(data) {
-    console.log(`key: ${ethutil.bufferToHex(data.key)}`)
-    console.log(`Value: ${ethutil.bufferToHex(rlp.decode(data.value))}`)
-  })
-  .on('end', function() {
-    console.log('Finished reading storage.')
-  })
+  stream
+    .on('data', function(data) {
+      console.log(`key: ${ethutil.bufferToHex(data.key)}`)
+      console.log(`Value: ${ethutil.bufferToHex(rlp.decode(data.value))}`)
+    })
+    .on('end', function() {
+      console.log('Finished reading storage.')
+    })
 })
 ```
 
 # API
+
 [./docs/](./docs/index.md)
```

Comment:
Did you generally leave out doc generation on purpose to keep this on the released version?

Reply:
Yeah, I remember something like this discussed somewhere :) plus they will most likely change in the next PRs